application/blueprint/services/continuous_planning_service.py

```
@@ -293,21 +293,46 @@ async def _generate_precise_macro_plan(
+        content_parts = []
+        async for chunk in self.llm_service.stream_generate(prompt, config):
+            content_parts.append(chunk)
+        
+        response = "".join(content_parts)
         updates = self._parse_llm_response(response)
-        self._merge_precise_structure_updates(
-            skeleton=skeleton,
-            updates=updates,
+
+        # 评估规划质量
+        elapsed_time = time.time() - start_time
```

Ruff (0.15.10)

[error] Undefined name `time`

(F821)